pkg/util/iptables/iptables.go

```diff
@@ -49,6 +51,18 @@ type Interface interface {
 	DeleteRule(table Table, chain Chain, args ...string) error
 	// IsIpv6 returns true if this is managing ipv6 tables
 	IsIpv6() bool
+	// Save calls `iptables-save` for table.
+	Save(table Table) ([]byte, error)
+	// SaveAll calls `iptables-save`.
+	SaveAll() ([]byte, error)
```

Comment:
These really should have unit tests.  I will accept a TODO as long as you promise to do them :)

The tests won't be hard, but they will ensure that what happens is what we expect.